src/resources/unlimitedCompany.ts

```diff
@@ -72,7 +72,7 @@ export const UnlimitedCompanyResource: ResourceIF = {
   },
   reviewAndConfirm: {
     completingPartyStatement: {
-      certifyStatementHeader: 'the Completing Party, have examined the Unlimited Company ' +
+      certifyStatementHeader: 'the Completing Party, have examined the Company ' +
```

Comment:
Wait a minute. Is this the same blurb for all company types now? If so, should the blurb be part of the component instead of in the resource file?

Reply:
Some bullets are the same atm yes, but not all.